crt_portal/cts_forms/views.py

```diff
@@ -41,7 +54,7 @@ def IndexView(request):
             "report_protected_classes": p_class_list
         })
 
-    return render_to_response('forms/index.html', {'data_dict': data, 'page_format': page_format})
+    return render_to_response('forms/index.html', {'data_dict': data, 'page_format': page_format, 'page_args': page_args})
```

Comment:
Would it be clearer to rename `page_args` to `page_sort_args`, `sort_args`, or `page_sorts`, or something else that mentions sorts? All of the arguments stored here are related to sorts, as I'm reading things. 

The `page_format` variable also holds args being sent to the page, so a more specific variable name might make the code clearer to read.

Reply:
Any suggestions for names?

Reply:
Left a few suggestions in the comment above, how about `sort_args`?

Reply:
The page_args aren't just for sorting, they currently have `per_page` and `sort` args. We will have more args when we have filters  like `start_date=20180515`, `end_date=20180518`, `status=new`, etc. These are all the args we need to make sure links on the page keep all the arguments that the viewer applied to the current page.

Reply:
Ah, I see, that wasn't clear to me at first — thanks for the explanation! 

We're using three variables to send data to the page: `data_dict`, `page_format`, and `page_args`. As a dev, I don't have a quick/easy way to understand the shape of the variables and the difference between `page_format` and `page_args` in particular when debugging or enhacning the view all table. If a variable rename isn't a good option, maybe a code comment could help, something like:

```
# page_args is for this kind of data
# 
# shape:
# {
#   ...
# }

# page_format is for this kind of data
# 
# shape:
# {
#   ...
# }
``` 

This is purely a suggestion, feel free to implement or merge the PR as-is. Hope this is helpful!